core/src/main/java/org/opensearch/sql/ast/tree/Relation.java

```diff
@@ -95,6 +95,20 @@ public String getTableNameOrAlias() {
     return (alias == null) ? getTableName() : alias;
   }
 
+  /**
+   * Return alias.
+   *
+   * @return alias.
+   */
+  public String getAlias() {
+    return alias;
+  }
+
+  public List<QualifiedName> getTableQualifiedNames() {
```

Comment:
why this function is required? does getTableName() enough?

Reply:
could we refactor the code and remove
* getCatalogName()
* getFullyQualifiedTableNameWithCatalog()
And revert 
* getTableName() as https://github.com/opensearch-project/sql/blob/2.3/core/src/main/java/org/opensearch/sql/ast/tree/Relation.java